heron/proto/topology.proto

```diff
@@ -71,9 +71,10 @@ message Config {
 message Component {
   // The name of the component. Like Tail-FlatMap
   required string name = 1;
-  // Only one of the below two are set
+  // Only one of the below three are set
```

Comment:
Could we change "jave_class_name" and "jave_object" to a more generic name, rather than adding a new filed "python_class_name", which is not extensible.